modules/configuring-assumerole-acs.adoc

```diff
@@ -10,20 +10,15 @@ After configuring AssumeRole in ECR, you can integrate {product-title} with Amaz
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Under the *Image Integrations* section, select *Amazon Elastic Container Registry*.
-+
-The *Configure image integration* modal box opens.
+. Under the *Image Integrations* section, select *Amazon ECR*.
 . Click *New Integration*.
 . Enter the details for the following fields:
 .. *Integration Name*: The name of the integration.
 .. *Registry ID*: The ID of the registry.
-.. *Region*: The region for the registry.
-. If you have configured AssumeRole with container IAM leave the *Use container IAM role* option checked. Otherwise, uncheck the *Use custom IAM role* and enter the details for the following fields:
-.. *Access key ID*: The access key ID for the role.
-.. *Secret access key*: The secret access key for the role.
-. Select the *Use AssumeRole* checkbox.
-. Enter the details for the following fields:
+.. *Region*: The region for the registry; for example, `us-west-1`.
+. If you are using IAM, select *Use container IAM role*. Otherwise, clear the *Use custom IAM role* box and enter the *Access key ID* and *Secret access key*.
+. If you are using AssumeRole, select *Use AssumeRole* and enter the details for the following fields:
```

Comment:
```suggestion
. If you are using `AssumeRole`, select *Use AssumeRole* and enter the details for the following fields:
```
(there are a few other instances of this in other modules too)

Reply:
I actually looked at the project, and it doesn't use the tick marks with AssumeRole anywhere else, so I left it unmarked to be consistent.